src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -48,7 +48,8 @@
  * <ul>
  * <li>
  * Property {@code excludeClassesRegexps} - Specify user-configured regular
- * expressions to ignore classes.
+ * expressions to ignore classes. All excluded packages should end with a period,
+ * so it also appends a dot to a package name.
```

Comment:
Where did `All excluded packages should end with a period, so it also appends a dot to a package name.` come from?

Reply:
removed.